install/minikube-installation-guide.md

```diff
@@ -0,0 +1,240 @@
+
+# A failed result means it is not enabled:
+ssh: Process exited with status 1
+```
+
+## Installing Kata Containers
+
+You can now install the Kata Containers runtime components. You will need a local copy of some Kata
+Containers components to help with this, and then use the host `kubectl` (that Minikube has already
+configured for you) to deploy them:
+
+```sh
+$ git clone https://github.com/kata-containers/packaging.git
+$ cd packaging/kata-deploy
+$ kubectl apply -f kata-rbac.yaml
+$ kubectl apply -f kata-deploy.yaml
+```
+
+This installs the Kata Containers components into `/opt/kata` inside the Minikube node. It can take
+a few minutes for the operation to complete. You can check the installation has worked by checking
+the status of the `kata-deploy` pod. This can be accomplished by running the following:
+
+```sh
+$ podname=$(kubectl -n kube-system get pods -o=name | fgrep kata-deploy | sed 's?pod/??')
+# If 'sleep infinity' is running, the pod has completed its work.
+$ kubectl -n kube-system exec ${podname} -- ps -ef | fgrep infinity
+root        49     1  0 10:55 ?        00:00:00 sleep infinity
+```
+
+> *NOTE:* This check only works for single node clusters, which is the default for Minikube.
+> For multi-node clusters, the check would need to be adapted to check `kata-deploy` had
+> completed on all nodes.
+
+## Enabling Kata Containers
+
+> **Note:** This step is only required for Minikube/Kubernetes versions <= 1.13. Since version
+> v1.14, the `RuntimeClass` is enabled by default. Performing this step on Kubernetes > v1.14 is
+> however benign.
+
+Now the Kata Containers components are installed in the Minikube node, you need to configure
+Kubernetes `RuntimeClass` to know when to use Kata Containers to run a pod.
+
+```sh
+$ curl https://raw.githubusercontent.com/kubernetes/node-api/master/manifests/runtimeclass_crd.yaml > runtimeclass_crd.yaml
+$ kubectl apply -f runtimeclass_crd.yaml
+```
+
+### Register the runtime
+
+Now register the `kata qemu` runtime with that class:
+
+```sh
+$ cd packaging/kata-deploy/k8s-1.14
+$ kubectl apply -f kata-qemu-runtimeClass.yaml
+runtimeclass.node.k8s.io/kata created
+runtimeclass.node.k8s.io/kata-qemu created
+```
+
+Kata Containers should now be installed and enabled in the Minikube cluster.
+
+## Testing Kata Containers
+
+Launch a container that is defined to run on Kata Containers. The enabling is configured by
+the following lines in the yaml file:
+
+```yaml
+    spec:
```

Comment:
Nit: You could remove the indent here so `spec:` is in column 1.

Reply:
but, as added to a pod YAML file, it will be indented as per the example, and almost definitely not flush to the LHS. I've added a link out to the actual k8s Runtime Class docs.